app/admin/user.rb

```diff
@@ -57,6 +55,39 @@ def password_params
     f.actions
   end
 
+  action_item :import_users, only: :index do
+    link_to 'ユーザーをインポートする', input_csv_admin_users_path
+  end
+
+  collection_action :input_csv, method: :get
+
+  collection_action :import_csv, method: :post do
+    unless params[:csv]
+      flash[:error] = 'CSVファイルを指定してください'
+      redirect_to :back and return
```

Comment:
__[rubocop]__
* Use `&&` instead of `and`. [:ref](https://github.com/bbatsov/ruby-style-guide#no-and-or-or)

<!-- Comment by [SideCI](https://sideci.com) -->